server/test/bookmark.spec.js

```diff
@@ -0,0 +1,88 @@
+import chai, { expect } from 'chai';
+import chaiHttp from 'chai-http';
+import users from '../utils/users';
+
+import app from '../..';
+
+chai.use(chaiHttp);
+
+let userToken;
+
+describe('Articles Endpoint /articles', () => {
+  before((done) => {
+    chai.request(app)
+      .post('/api/users/login')
+      .send({
+        user: {
+          username: users[0].username,
+          password: users[2].password1
+        }
+      })
+      .end((err, res) => {
+        userToken = res.body.user.token;
+        expect(res).to.have.status(200);
+        expect(res.body).to.be.an('object');
+        done();
+      });
+  });
+
+  it('it should create an article for bookmarking', (done) => {
```

Comment:
Why an `it` block to test creating an article. if you want to test for this article with this title `'how to code'` simply create another `before` for that.